mlir/tools/quantum-opt/quantum-opt.cpp

```diff
@@ -33,21 +33,9 @@ int main(int argc, char **argv)
     mlir::registerPass(catalyst::createQuantumConversionPass);
 
     mlir::DialectRegistry registry;
-    registry.insert<mlir::func::FuncDialect>();
-    registry.insert<mlir::arith::ArithDialect>();
-    registry.insert<mlir::scf::SCFDialect>();
-    registry.insert<mlir::LLVM::LLVMDialect>();
-    registry.insert<mlir::memref::MemRefDialect>();
-    registry.insert<mlir::tensor::TensorDialect>();
-    registry.insert<mlir::linalg::LinalgDialect>();
-    registry.insert<mlir::index::IndexDialect>();
-    registry.insert<mlir::bufferization::BufferizationDialect>();
+    mlir::registerAllDialects(registry);
```

Comment:
How would it affect the binary size of the tool and catalyst wheels in general? :thinking: 

It's indeed fixing issue #64 by registering all built-in dialects.

Reply:
I don't think binary size would change, since the dialects are still present in the tool (we can and have always been able to generate and manipulate ops from all those dialects, what this does is allow all those dialects to be parsed from an `.mlir` input file). But I haven't verified this.